scripts/new_project.kts

```diff
@@ -225,6 +240,16 @@ object NewProject {
         println("\n${message}\n")
         System.exit(exitCode)
     }
+
+    private fun String.uppercaseEveryFirstCharacter(): String {
+        return this.split(SPACE_SEPARATOR).joinToString(separator = SPACE_SEPARATOR) { string ->
```

Comment:
Hmm, interesting. Why do we need to split and then join the string before calling `replaceFirstChar`. I'm looking for this example https://tedblob.com/kotlin-string-first-character-uppercase/

Reply:
@luongvo The `capitalize` is deprecated and we are doing the `title case` which is transforming the string from `my project` to `My Project`, not `My project`.

Reply:
@luongvo Any last remarks? 😁

Reply:
Thank you for your all detailed explanations 👍